Integración Certificado votación

camilo1635 opened the pull request.

javierra218 committed:
feat: Modulo experto y línea de investigación

Se agregan módulos  de gestión de expertos y líneas de investigación. Se había venido trabajando en la rama "integracion-modulo-expertos", esto se hace para cargar los nuevos estilos.

javierra218 committed:
fix: Actualizar estilos de formulario de carga de documento de identificación

javierra218 committed:
feat: Agregar opción para mostrar/expertos inactivos en la bandeja de expertos.
TODO: funcionalidad para habilitarlo

javierra218 committed:
feat: actualizar modelos Categoria y Experto

- Actualizar modelos Categoria y Experto
- Agregar endpoint de lineas-investigacion/categoria/listar/ACTIVO

refactor: BandejaExpertosComponent

- Implementar método cambiarEstadoExperto
- Refactorizar método habilitarExperto

feat: implementar funciones CRUD en BandejaCategoriasComponent

- Implementar funciones CRUD (Crear, Leer, Actualizar, Eliminar) en BandejaCategoriasComponent

javierra218 committed:
feat: actualizar modelos Categoria y Experto

javierra218 committed:
feat: actualizar modelos Categoria y Experto

Refactorizar BandejaExpertosComponent y agregar funciones CRUD en BandejaCategoriasComponent

javierra218 committed:
feat: agregar filtro global en la tabla de categorías

Se agregó un filtro global en la tabla de categorías para buscar por nombre y observación.

javierra218 committed:
feat: Actualizar modelos Categoria y Experto

Refactorizar BandejaExpertosComponent y agregar funciones CRUD en BandejaCategoriasComponent

javierra218 committed:
feat: Actualizar modelos de Categoría y Experto, y refactorizar componentes

Se actualizaron los modelos de Categoría y Experto para mejorar su funcionalidad. Además, se refactorizaron los componentes BandejaExpertosComponent y BandejaCategoriasComponent para agregar funciones CRUD. También se agregó un filtro global en la tabla de categorías para buscar por nombre y observación.

javierra218 committed:
chore: Refactorize componentes "activo e inactivo" utilizando selectButton en lugar de switch

javierra218 committed:
feat: Actualizar propiedad "observacion" a "descripcion" en los modelos Categoria y Linea

Se actualizó la propiedad "observacion" a "descripcion" en los modelos Categoria y Linea para mejorar la consistencia y claridad del código. También se agregó la propiedad "descripcion" en los componentes BandejaCategoriasComponent y BandejaLineasComponent para reflejar este cambio.

javierra218 committed:
chore: Actualizar claves excluidas en fieldmap.ts

javierra218 committed:
Se cambiaron nombres a algunas variables para hacer el codigo mas descriptivo

javierra218 committed:
feat: Eliminar componentes de información financiación, información origen y información usuario

Se eliminaron los componentes de información financiación, información origen y información usuario, ya que no se utilizaban en el proyecto.

feat: Eliminar componentes de información financiación, información origen y información usuario

javierra218 committed:
feat: Actualizar componente BandejaExpertosComponent

Refactorizar componente BandejaExpertosComponent para mejorar funcionalidad y agregar función de agregar experto.

javierra218 committed:
feat: Refactorizar componente CrearEditarExpertoComponent

Refactorizar componente CrearEditarExpertoComponent para mejorar la organización del código y agregar secciones separadas para la información personal, de títulos y de vinculación.

javierra218 committed:
feat: Mejorar el formato del componente CrearEditarExpertoComponent y agregar secciones separadas para la información personal, de títulos y de vinculación

javierra218 committed:
feat: Actualizar importaciones y declaraciones de módulos

- Añadir `BandejaLineasComponent` a gestion-lineas.module.ts.
- Añadir `BandejaCategoriasComponent` a gestion-categorias.module.ts.
- Añadir `BandejaExpertosComponent` y `CargarExpertosComponent` a gestion-expertos.module.ts.
- Eliminar importaciones redundantes en app.module.ts.
- Corregir carga diferida de `GestionCategoriasModule` en app-routing.module.ts

javierra218 committed:
feat: Actualizar tooltips y iconos en componentes de eliminación, ngOnDestroy

Se actualizaron los tooltips y los iconos en los componentes de eliminación (BandejaExpertosComponent, BandejaCategoriasComponent y BandejaLineasComponent) para reflejar la acción de desactivar en lugar de eliminar.

se agrego metodo ngOnDestroy()

camilo1635 committed:
integración inicial solicitud certificado de votación

camilo1635 committed:
Merge branch 'feature/develop-testing' into certificado-votacion

camilo1635 committed:
strategy

camilo1635 committed:
Merge branch 'feature/develop-testing' into certificado-votacion

camilo1635 committed:
Integración envió de solicitud certificador de votación

jorgtunubala reviewed (changes requested):
Necesitamos esta rama sin nuevos cambios porque se tienen pruebas pendientes de despliegue y esta integración nos puede afectar

camilo1635 committed:
ajustes tramite solicitud certificado de votacion

camilo1635 committed:
gestion inicial de certificados

camilo1635 committed:
Merge branch 'integracion-modulo-experto' into prueba-integracion-experto

camilo1635 committed:
Integracion experto finalizada

camilo1635 committed:
mostrar y ocultar boton aprobar

camilo1635 committed:
ajuste componente gestion certificados de votacion

se creo un servicio principalmente para recibir el archivo .zip del back con los documentos PDF

camilo1635 committed:
Merge branch 'prueba-integracion-experto' into certificado-votacion
